tests/pytorch/mpops/test_edgewise.py

```diff
@@ -0,0 +1,201 @@
+import random
```

Comment:
Why do we need pytorch in the path?

Reply:
Because currently DGL's unit tests are split into two sets:
* All unit tests under `tests/compute` are framework-agnostic unit tests (can run for PyTorch/MXNet/TensorFlow).
* All unit tests under `tests/pytorch` are pytorch-specific unit tests (only runnable for Pytorch).

In the future, as the part of the effort to make DGL more PyTorch-friendly, we should gradually add tests only to `tests/pytorch` and eventually remove the `pytorch` subfolder (dropping the support of the other two).